core/mixins/_retina-bg-image.scss

```diff
@@ -17,11 +17,13 @@
  */
```

Comment:
This sounds good too. Happy for you to do the docs if you want. I won't get to them tonight now